model/export/implementation/LogEntryCsvExporter.php

```diff
@@ -47,16 +49,29 @@ public function __construct()
     /**
      * @param array $params
      * @return mixed
+     * @throws \Exception
      */
     public function export(array $params = [])
     {
-        $from = (new DateTimeImmutable())
-            ->sub(new DateInterval($this->loggerService->getOption(LoggerService::OPTION_EXPORTABLE_PERIOD)))
```

Comment:
is it OK that you never use OPTION_EXPORTABLE_PERIOD after that change?

Reply:
Yes. Instead of EXPORATBLE_PERIOD we will use `from` and  `to` dates.